### PR TITLE
Add PAdES sign support

### PR DESCRIPTION
**Pull request by a5i**
This change is [Reviewable](https://reviewable.io/reviews/unidoc/unipdf/374)

### CODE REVIEW

File: model/appender_test.go [hunk 1]
```diff
@@ -1819,3 +1823,149 @@ func TestSignatureAppearanceWithTimestamp(t *testing.T) {
...
+
+	validateFile(t, "./testdata/dss/siwa.pdf")
+	validateFile(t, "./testdata/dss/Test.signed_Certipost-2048-SHA512.extended-LTA.pdf")
+	validateFile(t, "./testdata/dss/TEST2_signed_with_zero_policy_hash.pdf")
+	validateFile(t, "./testdata/dss/TestToSignPDFSHA256_TST_SIG_NOT_FOUND.pdf")
+
+}
+
+func TestAppenderSignPAdES(t *testing.T) {
+	pass := os.Getenv("TINYCERT_USER_PASS")
```
Comment (gunnsth) [inline model/appender_test.go:1854]: is it possible to automate this test better?
or we can define some static/test credentials?

File: model/sighandler/sighandler_pades.go [hunk 2]
```diff
@@ -0,0 +1,407 @@
+package sighandler
```
Comment (gunnsth) [inline model/sighandler/sighandler_pades.go:1]: add license header

File: model/sighandler/sighandler_pades.go [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+func (a *etsiPAdES) NewDigest(sig *model.PdfSignature) (model.Hasher, error) {
+	return bytes.NewBuffer(nil), nil
+}
+
+// Validate validates PdfSignature.
+func (a *etsiPAdES) Validate(sig *model.PdfSignature, digest model.Hasher) (model.SignatureValidationResult, error) {
+	return a.ValidateEx(sig, digest, nil)
+}
+
+// ValidateEx validates PdfSignature with additional information.
```
Comment (gunnsth) [inline model/sighandler/sighandler_pades.go:335]: what additional information?

File: model/testdata/dss/LICENSE [hunk 4]
```diff
@@ -0,0 +1,502 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (gunnsth) [inline model/testdata/dss/LICENSE:1]: what file/s is/are under this license?

Comment (a5i) [reply model/testdata/dss/LICENSE:7]: Added a header

Comment (gunnsth) [reply model/testdata/dss/LICENSE:7]: @a5i It applies to all files under this folder testdata/dss ?

File: model/appender_test.go (hunk 1 above)
Comment (a5i) [reply model/appender_test.go]: I changed the certificate password to default.

**Pull request closed without merging**